Modules/DevTools/BusinessCentralPerformanceToolkit/src/BCPTLines.Page.al

```diff
@@ -242,10 +242,9 @@ page 149004 "BCPT Lines"
             exit(true);
         if Rec."BCPT Code" <> BCPTHeader.Code then
             if BCPTHeader.Get(Rec."BCPT Code") then;
-        if Rec."Min. User Delay (ms)" = 0 then
```

Comment:
The idea here was that maybe the user had entered some value before the line gets inserted (delayed insert).
An unconditional assignment would probably fit better in the OnNewRecord trigger, I think.

Reply:
I've moved the assignments to OnNewRecord trigger. There should be no issue with the default being 0 because of minValue property being set on the table. The validation will always run when it is changed on the page.

Reply:
Makes sense!